### PR TITLE
FIX Handle symfony deprecation warning

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/9955

The `error_get_last()` in the `shutdown()` function was causing an unrelated and otherwise harmless deprecation warning on a symfony component to break the upload field, including on environments which were set to 'live' which wouldn't otherwise give deprecation warnings.

### CODE REVIEW

File: src/BulkTools/HTTPBulkToolsResponse.php [hunk 1]
```diff
@@ -369,7 +369,7 @@ protected function outputBody()
     public function shutdown()
     {
         $error = error_get_last();
-        if ($error !== null ) {
+        if ($error !== null && $error['type'] !== E_USER_DEPRECATED) {
```
Comment (maxime-rainville) [inline src/BulkTools/HTTPBulkToolsResponse.php:372]: There's a lot of other warnings and notices being thrown aside from the deprecation warning. Even so, in a scenario where multiple errors are thrown, you'll miss all of them if the last one is a deprecation warning.

You're also exposing an internal error message even in a live environment, which is a security risk.

It's not even clear to me the the error message ends up being displayed in a useful way to the user in the front end.

Basically, this approach is incredibly shaky and I think we would be better off gutting the shut down method instead.

**Review by maxime-rainville — CHANGES_REQUESTED** [review]

File: src/BulkTools/HTTPBulkToolsResponse.php (hunk 1 above)
Comment (kinglozzer) [reply src/BulkTools/HTTPBulkToolsResponse.php:372]: https://github.com/colymba/GridFieldBulkEditingTools/pull/205 😄

**Comment by emteknetnz** [comment]
Closed in favour of https://github.com/colymba/GridFieldBulkEditingTools/pull/205
